src/Data/ExtractShardKey.ts

```diff
@@ -168,3 +170,26 @@ export async function extractKeyFromTx(receiptTx: any): Promise<string> {
   // return transformedSourceKey // executionShardKey
   return 'placeholder'
 }
+
+export function getTransactionObj(tx: any): Transaction[TransactionType.Legacy] | Transaction[TransactionType.AccessListEIP2930] {
+  if (!tx.raw) throw Error('fail')
```

Comment:
why so generic? is it copy pasted method?

Reply:
yes